google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ClientSideStatementValueConverters.java

```diff
@@ -85,6 +88,43 @@ public Boolean convert(String value) {
     }
   }
 
+  /** Converter from string to {@link Boolean} */
+  static class PgBooleanConverter implements ClientSideStatementValueConverter<Boolean> {
+
+    public PgBooleanConverter(String allowedValues) {}
+
+    @Override
+    public Class<Boolean> getParameterClass() {
+      return Boolean.class;
+    }
+
+    @Override
+    public Boolean convert(String value) {
+      if (value == null) {
+        return null;
+      }
+      if (value.length() > 1
+          && ((value.startsWith("'") && value.endsWith("'"))
```

Comment:
I think Double Quotes(`""`) are used to identify objects inside the database like column names, table names etc. So, `"true"` shouldn't work